src/Roave/SecurityAdvisories/AdvisorySources/GetAdvisoriesFromGithubApi.php

```diff
@@ -122,7 +122,15 @@ private function getAdvisories() : array
         $cursor     = '';
 
         do {
-            $response        = $this->client->sendRequest($this->getRequest($cursor));
+            $response = $this->client->sendRequest($this->getRequest($cursor));
+            /** @psalm-var array{
```

Comment:
We really should revive https://github.com/vimeo/psalm/issues/2781 about this

Reply:
Shouldn't this be a feature from Assert lib that could look like:
Assert::allKeyExistRecursive($array, ['key1' => ['deepKey2', 'deepKey3'], 'key2') ?

This may be easier to explain once we have a concrete way to check that sort of structures?

Reply:
Yeah, that's kinda what I wanted to have, but vimeo/psalm#2781 seemed to become a "won't fix" :-S